src/components/popover/examples/Popover.vue

```diff
@@ -159,6 +158,10 @@ export default {
     popupHandler(e) {
       console.log(e);
     },
+    closedCustomHandler(e) {
```

Comment:
Every popover with a custom trigger will need to have to listen for the "closed" event after the popover is closed, because popovers with custom triggers have no control over the state of the button that controls them. This is actually broken in the docsite too and will need an extra bullet point to let users know that they need to add an event handler in order for it to work properly